### ISSUE TITLE
About SEED-X  Visual De-tokenization pretraining code

### ISSUE THREAD

**Issue by qxhhou** [AILab-CVC/SEED-X#7]
Thanks for your great work!
Will you share the visual de-tokenization training code? 
I want to tune the de-tokenization model (base on SDXL).

**Comment by geyuying** [comment]
Hi, we are not going to release the visual de-tokenization training code at present :grin:.
